blogposts/2021/release-3.34.md

```diff
@@ -0,0 +1,45 @@
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.34/sourcegraph-3-34-release.png
+changelogItems:
+  - description: 'Added a help page providing search tips if a search returns no results, making it easier to troubleshoot your query.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/26154
+    category: Search
+  - description: 'In order to reduce load on the Gitserver service on installations with a large number of repositories, the search indexer only polls repositories that have been marked as changed. If you notice index staleness you can try disabling by setting the environment variable `SRC_SEARCH_INDEXER_EFFICIENT_POLLING_DISABLED` on `sourcegraph-frontend`.'
+    url: https://github.com/sourcegraph/sourcegraph/issues/27058
+    category: Search
+  - description: 'To help site admins debug issues with webhook delivery, we have added logging of incoming Batch Changes webhooks. By default, sites without encryption will log webhooks for three days and sites with encryption will not log webhooks.'
+    url: https://docs.sourcegraph.com/admin/config/batch_changes#incoming-webhooks
+    category: Admin
+  - description: 'To allow writing to the `.github` directory in repositories, Batch Changes now requests the `workflow` scope on GitHub personal access tokens. If you have already configured a GitHub PAT for Batch Changes, we suggest adding this scope.'
+    url: https://docs.sourcegraph.com/batch_changes/how-tos/configuring_credentials
+    category: Batch Changes
+  - description: "To increase search speed, the search UI's repository count and the GraphQL API's `search().repositories`/`search().repositoriesCount` have changed semantics from being the set of searchable repositories to the set of repositories with matches. In a future release, we’ll introduce separate fields for searchable repositories."
+    url: https://github.com/sourcegraph/sourcegraph/issues/26995
+    category: Search
+  - description: 'Version contexts and `repogroup` filters (deprecated in v3.33) have been removed in favor of search contexts.'
+    url: https://about.sourcegraph.com/blog/introducing-search-contexts/
+    category: Search
+  - description: 'Saved search notifications (deprecated in v3.31.0) have been removed in favor of code monitoring.'
+    url: https://docs.sourcegraph.com/code_monitoring
+    category: Search
+---
+
+Sourcegraph 3.34 is now available! Here are some highlights from this release:
+
+## Troubleshoot your searches with the "no results" page
+
+Searches that return no results can result in a confusing experience. When this occurs, we now display helpful tips on how to use search, examples of how to solve common query issues, and links to tutorial videos addressing common use cases.
+<img class="blog-image" title="Find implementations for Go" alt="Find implementations for Go" src="https://storage.googleapis.com/sourcegraph-assets/blog/3.34/no_results_page.png">
```

Comment:
```suggestion
<img class="blog-image" title="No results page" alt="This image shows the new no results page, with resources available to troubleshoot your search when no results are returned." src="https://storage.googleapis.com/sourcegraph-assets/blog/3.34/no_results_page.png">
```

cc @benvenker the alt and title tags here seemed wrong so I updated them. If these are wrong too please follow up with a correction.